lib/core/msdf-sys/build.rs

```diff
@@ -0,0 +1,39 @@
+extern crate download_lp;
+
+mod msdfgen_wasm {
+
+    pub const VERSION  : &str = "v1.0";
+    pub const FILENAME : &str = "msdfgen_wasm.js";
+
+    pub fn download() {
```

Comment:
I know it may sound very verbose, but before merging to master each top-level public function has to has a documentation. This way we can generate nice docs to our libraries for new people joining the porject. See this guide: https://github.com/luna/enso/blob/master/doc/rust-style-guide.md#documentation-comments

Reply:
Technically it is not public because:
a) it's a build script - not designed to be imported anywhere
b) it's a private module of build script